fixtures/devtools/iframe/iframe-same-origin-sandbox.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
```

Comment:
Maybe you can add your case like this and push a PR ?

Reply:
This is a good suggestion!